README.md

```diff
@@ -340,6 +340,7 @@ const options = {
     body: {
       noMatch: "Sorry, no matching records found",
       toolTip: "Sort",
+      columnHeaderTooltip: "Sort for `${column.label}`"
```

Comment:
Can we use a function in the example, since that's how people would have to implement it? So more like `columnHeaderTooltip: column =>` `Sort for ${column.label}`.

Reply:
Done.